### PR TITLE
Update daos-overview.md

### PR DESCRIPTION
**Pull request by yghadar**
rewrite the text as the font was screwed in the website

### CODE REVIEW

**Comment by cjknight** [comment]
I don't believe this fixes the formatting issue. And now "Job" is misspelled. There is also "env" further below that is formatted as a section heading. Could place all code snippets with 3-backtick codeblocks like in other pages.

File: docs/aurora/data-management/daos/daos-overview.md [hunk 1]
```diff
@@ -20,162 +18,175 @@ The pool will be set to allow anyone in the project unix group to access the poo
...
+  Total size: 200 TB
+  Free: 194 TB, min:244 GB, max:308 GB, mean:303 GB
+Rebuild done, 4 objs, 0 recs
+```
 
 ## Container
 The container is the basic unit of storage.
 A POSIX container can contain hundreds of millions of files, you can use it to store all of your data.
-You only need a small set of containers perhaps just one per major unit of project work.
+You only need a small set of containers; perhaps just one per major unit of project work is.
```
Comment (cjknight) [inline docs/aurora/data-management/daos/daos-overview.md:51]: Word missing? "project work is sufficient" ?

**Review by cjknight — APPROVED** [review]
Looks good!

File: docs/aurora/data-management/daos/daos-overview.md (hunk 1 above)
Comment (felker) [inline docs/aurora/data-management/daos/daos-overview.md:51]:
```suggestion
You only need a small set of containers; perhaps just one per major unit of project work is sufficient.
```

**Pull request merged**